tests/integration/api_team_test.go

```diff
@@ -58,7 +60,7 @@ func TestAPITeam(t *testing.T) {
 	session = loginUser(t, user.Name)
 	token = getTokenForLoggedInUser(t, session, auth_model.AccessTokenScopeAdminOrg)
 
-	org := unittest.AssertExistsAndLoadBean(t, &user_model.User{ID: 6})
+	org = unittest.AssertExistsAndLoadBean(t, &organization.Organization{ID: 6})
```

Comment:
Why `user` -> `organization`?

Reply:
> Why `user` -> `organization`?

The previous `org` is a `*User`, now since `org` has been declared, so here we should use `*Organization` as org.